crates/ruff_linter/src/rules/ruff/rules/ambiguous_unicode_character.rs

```diff
@@ -83,7 +83,7 @@ impl Violation for AmbiguousUnicodeCharacterString {
 /// spec recommends `GREEK CAPITAL LETTER OMEGA` over `OHM SIGN`.
 ///
 /// You can omit characters from being flagged as ambiguous via the
-/// [`allowed-confusables`] setting.
+/// [`lint.allowed-confusables`] setting.
```

Comment:
Oh I'm surprised that our doc generation script didn't warn about this one because it checks all options for validity.